packages/system/external-secrets-operator/values.yaml

```
@@ -0,0 +1,4 @@
+external-secrets:
+  bitwarden-sdk-server:
+    enabled: false  
```

yamllint

[error] trailing spaces

(trailing-spaces)